creator-node/src/app.js

```diff
@@ -43,6 +44,13 @@ app.use('/batch_cids_exist', batchCidsExistReqLimiter)
 app.use('/batch_image_cids_exist', batchCidsExistReqLimiter)
 app.use(getRateLimiterMiddleware())
 
+// Block content upload on these routes if app shutting down
+app.use('/track_content_async', ensureAppIsOnline)
+app.use('/tracks/metadata', ensureAppIsOnline)
+app.use('/tracks', ensureAppIsOnline)
+app.use('/image_upload', ensureAppIsOnline)
+// block creating users? creating playlists? these should not be heavy ops, besides the chain verification
```

Comment:
probably dont need to block creating users and playlists since these are not heavy ops compared to image/track upload